frontend/src/metabase/timelines/containers/TimelineModal/TimelineModal.tsx

```diff
@@ -19,12 +19,16 @@ const collectionProps = {
     Urls.extractCollectionId(props.params.slug),
 };
 
+const mapStateToProps = () => ({
+  archived: false,
+});
+
```

Comment:
Why do we need to hardcode `archived: false` here?

Reply:
I've made `archived` prop optional in `TimelineModal` with default to `false`.